src/Hashing/ArgonHashing.php

```diff
@@ -57,22 +57,22 @@ class ArgonHashing extends AbstractHashing
     /**
      * __construct.
      *
-     * @param (array) $options
+     * @param array $options
      *
      * @since 3.0.0
      */
-    public function __construct(array $options)
+    public function __construct(array $options = null)
```

Comment:
To keep the `$options` being null because we have to let `??` syntax be the expected behavior.

Reply:
yes you;re right

Reply:
you can fix so i can review PR

Reply:
@Lablnet, I've fixed that and let `$options` be the `null` because of the `??` syntax.